static/app/views/replays/detail/layout/index.tsx

```diff
@@ -60,8 +61,8 @@ function ReplayLayout({
 
   const content = (
     <ErrorBoundary mini>
-      <FluidPanel title={<FocusTabs />}>
-        <FocusArea />
+      <FluidPanel title={<FocusTabs />} bodyRef={containerRef}>
+        <FocusArea parentRef={containerRef} />
```

Comment:
ref shouldn't be used for more then one element. Is there a reason you're creating the ref here and passing it down a couple of levels vs just creating it in the consoleMessage file?

Reply:
I'm actually using it for one element, but I'm passing the ref of the element via props because I need the ref of the container as this is the one that has the scroll so I can later use it with `useCurrentItemScroller` hook

Reply:
You could create the ref in ConsoleMessage component and assign it to the ConsoleTable container in there instead of passing through these layout components

Reply:
I tried that but it wouldn't work as `ConsoleTable` is not the one with the scroll

Reply:
Ahhh okay I didn't realize it had to be on the scroll container

Reply:
Yeah tbh I'm not a big fan of passing the ref via props either, but I think we would need to refactor a bit how the tabs behave so we can do another workaround 🤔

Reply:
Each tab should be in charge of it's own scrolling. The network table, for example, controls it's own scrolling which is what makes it possible for the headers to be fixed. 
If fixing all the other tabs will makes things better here then we should do it. That change should happen sooner or later anyway.